timemachine/fe/mcgregor.py

```diff
@@ -128,6 +128,30 @@ def __init__(self, n_vertices, edges):
             for edge_idx in edges:
                 self.ve_matrix[vertex_idx][edge_idx] = True
 
+    def _verify_partially_connected(self, source, node_states, n_mapped_nodes):
+        """A fast BFS node generator, adapted from _plain_bfs in networkx."""
+        unseen = [1] * self.n_vertices
```

Comment:
nit: Here 1 means unseen and 0 means seen? Its a bit confusing with the constants that have 0, 1, 2 values, but have different meanings.

Reply:
nit: any reason to not use use `bool` here?

Reply:
will do True/False

Reply:
wow - using a bool here yielded a 50% speed-up